.github/ISSUE_TEMPLATE/2.feature.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Feature request
+about: Suggest a new idea to Chirpstack
+---
+
+<!--
+  We really appreciate your time effort in creating this issue, it's really value for the enhancement of the project.
+  Before diving into the details, make sure to check off the following:
+-->
+
+<!-- Your checkbox should look like this: [x] -->
+
+- [ ] I have searched the [issues](https://github.com/brocaar/chirpstack-gateway-bridge) of this repository and believe that this is not a duplicate.
+
+## Summary
+
+<!-- Describe how it should work. -->
```

Comment:
I think it would be good to add a section _What is the use-case?_: Describe what problem or use-case is solved with this feature.

Sometimes there are multiple ways to deal with a problem or use-case :) This will help to provide feedback / shape the implementation / make sure that it fits within the architecture. E.g. the MQTT token pull-request was a solution which was specific to Mosquitto. However the use-case was to have a feature which would enable the generation of credentials to authenticate gateways.

Reply:
You are right. I agree with this point.